Do not force BOOST_NO_CXX11_RVALUE_REFERENCES; it breaks Boost >= 1.87.
Based on the media-gfx/epsonscan2 ebuild from the slonko overlay.

diff --git a/src/CommonUtility/ESCommonTypedef.h b/src/CommonUtility/ESCommonTypedef.h
index 5a9d416..34f29a5 100644
--- a/src/CommonUtility/ESCommonTypedef.h
+++ b/src/CommonUtility/ESCommonTypedef.h
@@ -14,7 +14,6 @@
 
 #include <string>
 #include <sstream>
-#define BOOST_NO_CXX11_RVALUE_REFERENCES 1
 #include <boost/any.hpp>
 //#include <boost/unordered_map.hpp
 #include <unordered_map>
diff --git a/src/CommonUtility/Linux/CMakeLists.txt b/src/CommonUtility/Linux/CMakeLists.txt
index 61bfd34..9d93dce 100644
--- a/src/CommonUtility/Linux/CMakeLists.txt
+++ b/src/CommonUtility/Linux/CMakeLists.txt
@@ -19,7 +19,6 @@
 # プロジェクト名
 PROJECT(commonutility)
 
-add_definitions(-DBOOST_NO_CXX11_RVALUE_REFERENCES=1)
 
 # ビルドするためにインクルードするヘッダーファイルの場所
 INCLUDE_DIRECTORIES(
diff --git a/src/Controller/CMakeLists.txt b/src/Controller/CMakeLists.txt
index 0f86a32..794fd31 100644
--- a/src/Controller/CMakeLists.txt
+++ b/src/Controller/CMakeLists.txt
@@ -24,7 +24,6 @@ LINK_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/../CommonUtility/Linux)
 
 PROJECT(epsonscan2)
 
-add_definitions(-DBOOST_NO_CXX11_RVALUE_REFERENCES=1)
 
 INCLUDE_DIRECTORIES(
 ../
diff --git a/src/Controller/Src/KeysValues/Key.hpp b/src/Controller/Src/KeysValues/Key.hpp
index 0bb6a27..bff05b0 100644
--- a/src/Controller/Src/KeysValues/Key.hpp
+++ b/src/Controller/Src/KeysValues/Key.hpp
@@ -2,7 +2,6 @@
 #include "EpsonScan2.h"
 #include "KeyDataProvider.hpp"
 #include <ESCommonTypedef.h>
-#define BOOST_NO_CXX11_RVALUE_REFERENCES 1
 #include <boost/any.hpp>
 #include <memory>
 
diff --git a/src/Controller/Src/KeysValues/KeyMgr.hpp b/src/Controller/Src/KeysValues/KeyMgr.hpp
index 051aad0..bda69b0 100644
--- a/src/Controller/Src/KeysValues/KeyMgr.hpp
+++ b/src/Controller/Src/KeysValues/KeyMgr.hpp
@@ -1,5 +1,4 @@
 #pragma once
-#define BOOST_NO_CXX11_RVALUE_REFERENCES 1
 #include <ESCommonTypedef.h>
 #include "EpsonScan2.h"
 #include "KeyDataProvider.hpp"
diff --git a/src/ES2Command/Linux/CMakeLists.txt b/src/ES2Command/Linux/CMakeLists.txt
index 8f3eb84..8981936 100644
--- a/src/ES2Command/Linux/CMakeLists.txt
+++ b/src/ES2Command/Linux/CMakeLists.txt
@@ -19,7 +19,6 @@
 # プロジェクト名
 PROJECT(es2command)
 
-add_definitions(-DBOOST_NO_CXX11_RVALUE_REFERENCES=1)
 
 if(COMMAND cmake_policy)
       cmake_policy(SET CMP0003 NEW)  
diff --git a/src/SaneWrapper/Linux/CMakeLists.txt b/src/SaneWrapper/Linux/CMakeLists.txt
index 62c06af..b222bbc 100644
--- a/src/SaneWrapper/Linux/CMakeLists.txt
+++ b/src/SaneWrapper/Linux/CMakeLists.txt
@@ -19,7 +19,6 @@
 # プロジェクト名
 PROJECT(sane-epsonscan2)
 
-add_definitions(-DBOOST_NO_CXX11_RVALUE_REFERENCES=1)
 
 if(COMMAND cmake_policy)
       cmake_policy(SET CMP0003 NEW)  
diff --git a/src/ScanSDK/CMakeLists.txt b/src/ScanSDK/CMakeLists.txt
index af5f907..ce26037 100644
--- a/src/ScanSDK/CMakeLists.txt
+++ b/src/ScanSDK/CMakeLists.txt
@@ -21,7 +21,6 @@ cmake_minimum_required (VERSION 3.10)
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
-add_definitions(-DBOOST_NO_CXX11_RVALUE_REFERENCES=1)
 
 ADD_LIBRARY(epsonscansdk SHARED
 #add_executable(epsonscansdk
diff --git a/src/Standalone/CMakeLists.txt b/src/Standalone/CMakeLists.txt
index 002f18f..54c34c4 100644
--- a/src/Standalone/CMakeLists.txt
+++ b/src/Standalone/CMakeLists.txt
@@ -28,7 +28,6 @@ find_package(Qt5Core REQUIRED)
 find_package(Qt5Gui REQUIRED)
 find_package(Qt5Widgets REQUIRED)
 
-add_definitions(-DBOOST_NO_CXX11_RVALUE_REFERENCES=1)
 
 INCLUDE_DIRECTORIES(
 ../
diff --git a/src/Standalone/defaultsettings.cpp b/src/Standalone/defaultsettings.cpp
index fdb13f6..6010c85 100644
--- a/src/Standalone/defaultsettings.cpp
+++ b/src/Standalone/defaultsettings.cpp
@@ -1,7 +1,6 @@
 #include "defaultsettings.h"
 
 #include <fstream>
-#define BOOST_NO_CXX11_RVALUE_REFERENCES 1
 #include <boost/any.hpp>
 #include <utils/PathUtils.h>
 
diff --git a/src/Standalone/lastusedsettings.cpp b/src/Standalone/lastusedsettings.cpp
index 908b5ce..ad592cf 100644
--- a/src/Standalone/lastusedsettings.cpp
+++ b/src/Standalone/lastusedsettings.cpp
@@ -17,7 +17,6 @@
 //  lastusedsettings.cpp -- template and derived ESC/I ESC/I-2 protocol commands
 
 #include <fstream>
-#define BOOST_NO_CXX11_RVALUE_REFERENCES 1
 #include <boost/any.hpp>
 #include <utils/PathUtils.h>
 #ifndef NO_QTMODE
